src/Network/Packet.h

```diff
@@ -80,7 +80,7 @@ namespace OP2Internal
 
 	struct TransportLayerHeader
 	{
-		TransportLayerCommandType commandType;
+		TransportLayerCommand commandType;
```

Comment:
I had considered changing this name from commandType to just command.

Reply:
That's maybe not a bad idea. I'd do a quick check for other similarly named fields in other related structs first though.